### PR TITLE
Fix English 1

### PR DESCRIPTION
**Pull request by Seaqn**
fixed the notifications to be more understandable

### CODE REVIEW

**Comment by darknight2590** [comment]
and truckerjob

File: client/main.lua [hunk 1]
```diff
@@ -185,7 +185,7 @@ Citizen.CreateThread(function()
                                 QBCore.Functions.TriggerCallback('qb-garbagejob:server:CheckBail', function(DidBail)
                                     if DidBail then
                                         BringBackCar()
-                                        QBCore.Functions.Notify("You have $250,- deposit returned!")
+                                        QBCore.Functions.Notify("$250 dollar deposit was returned!")
```
Comment (papabendi) [inline client/main.lua:188]: QBCore.Functions.Notify("Your $250 deposit has been returned!")

File: client/main.lua [hunk 2]
```diff
@@ -210,11 +210,11 @@ Citizen.CreateThread(function()
                                             GarbageLocation = 1
                                             IsWorking = true
                                             SetGarbageRoute()
-                                            QBCore.Functions.Notify("You have $250,- deposit paid!")
-                                            QBCore.Functions.Notify("You have started working, location marked on GPS!")
+                                            QBCore.Functions.Notify("$250 dollar deposit was taken out of your bank account!")
```
Comment (papabendi) [inline client/main.lua:213]: QBCore.Functions.Notify("The $250 deposit has been taken out of your bank account!")

File: client/main.lua [hunk 3]
```diff
@@ -210,11 +210,11 @@ Citizen.CreateThread(function()
...
+                                            QBCore.Functions.Notify("You have started work, first location marked on GPS!")
```
Comment (papabendi) [inline client/main.lua:214]: QBCore.Functions.Notify("You have started work. Your first location has been marked on the GPS!")

File: client/main.lua [hunk 4]
```diff
@@ -210,11 +210,11 @@ Citizen.CreateThread(function()
...
                                         end, coords, true)
                                     else
-                                        QBCore.Functions.Notify("You have not enough money for the deposit.. Deposit costs are $1000,-")
+                                        QBCore.Functions.Notify("You dont have enough money to pay the deposit.. Deposit cost is $250 dollars")
```
Comment (papabendi) [inline client/main.lua:217]: QBCore.Functions.Notify("You dont have enough money to pay the deposit.. The deposit cost is $250")

**Review by papabendi — CHANGES_REQUESTED** [review]
Although, the original costs are $1000, could be better to have it referring to a variable for the cost. That way it could be changed.

**Pull request closed without merging**